src/promptflow/dev-connections.json.example

```diff
@@ -9,6 +9,16 @@
     },
     "module": "promptflow.connections"
   },
+  "azure_open_ai_aad": {
+    "type": "AzureOpenAIConnection",
+    "value": {
+      "api_base": "aoai-api-endpoint",
+      "api_type": "azure",
+      "auth_mode": "meid_token",
```

Comment:
Discuss shall we add change log for auth_mode, or just waiting for tools ready and announce?